src/praisonai/praisonai/cli/features/setup/handler.py

```
@@ -288,27 +288,57 @@ def secure_opener(path, flags):
+        """Save configuration to unified config.yaml."""
         config_file = praison_home / "config.yaml"
         
+        # Read existing config if it exists
+        existing_config = {}
+        if config_file.exists():
+            try:
+                import yaml
+                with open(config_file, 'r') as f:
+                    existing_config = yaml.safe_load(f) or {}
+            except Exception:
+                pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)